packages/kolibri-tools/lib/cli.js

```diff
@@ -641,6 +641,33 @@ _addPathOptions(program.command('i18n-profile'))
     profileStrings(pathInfo, options.ignore, options.outputFile, options.verbose);
   });
 
+// I18N Ditto Audit
+_addPathOptions(program.command('i18n-audit'))
+  .option(
+    '--output-file <outputFile>',
+    'File path and name to which to write out the audit to',
+    filePath
```

Comment:
I was having a hard time figuring out why this `filePath` token was here or what it did. I found that it's a function -- is this basically a callback function that takes in the user input as its param?

Reply:
Yes, it's a validation function - unfortunately commander allows some not very intuitive argument overloading, and if you pass a callback as the third argument, it uses it to validate the argument and returns the value from the function instead.